web-client/src/presenter/actions/navigateToMessageDetailAction.test.js

```diff
@@ -0,0 +1,33 @@
+import { navigateToMessageDetailAction } from './navigateToMessageDetailAction';
+import { presenter } from '../presenter-mock';
+import { runAction } from 'cerebral/test';
+
+describe('navigateToMessageDetailAction', () => {
+  let routeStub;
+  const CASE_ID = 'fc1b424c-885f-450d-baee-b1965fd67149';
+  const PARENT_MESSAGE_ID = '14ccdbd2-896a-4a28-a603-b8b3030c84f7';
+
+  beforeAll(() => {
+    routeStub = jest.fn();
+
+    presenter.providers.router = {
+      route: routeStub,
+    };
+  });
+
+  it('navigates to message detail page without parent message id', async () => {
```

Comment:
@ericsorenson `without parent message id` makes me think the path param for parent message id shouldn't be passed into the url, but it looks like it is?

Reply:
The test description was incorrect. It has been updated!